docs/pages/guides/monorepos.mdx

```diff
@@ -62,16 +62,20 @@ Before we create our app, we have to create the **apps** directory. This directo
 
 After copying or creating the first app, run `yarn` to check for common warnings.
 
-#### Modify the Metro config
+#### Configure Metro
 
-Expo's Metro config has monorepo support for bun, npm, yarn, and pnpm. You don't have to manually configure Metro when using monorepos if you use the config from [`expo/metro-config`](/guides/customizing-metro/). If that's the case, you can skip this step.
+Expo's Metro config has monorepo support for bun, npm, yarn, and pnpm. By default, if you use Expo CLI and do not have a **metro.config.js** file, you be will using the config from `expo/metro-config`, and you can proceed to the next step. 
```

Comment:
```suggestion
Expo's Metro config has monorepo support for bun, npm, yarn, and pnpm. By default, if you use Expo CLI and do not have a **metro.config.js** file, you will be using the config from `expo/metro-config`. You can skip this and proceed to the next step. 
```

Grammatical typo and let's add the context about skipping this step for clarity.